yarn.lock

```diff
@@ -2995,27 +2987,6 @@
     node-properties-parser "^0.0.2"
     properties-to-json "^0.1.7"
 
-"@okta/okta-auth-js@^6.0.0":
-  version "6.0.0"
-  resolved "https://registry.yarnpkg.com/@okta/okta-auth-js/-/okta-auth-js-6.0.0.tgz#5ec34f7d92f6c91a3efeef8a6441175693b6fa74"
-  integrity sha512-ivvAJJKxYKIWIP1dP7Fz8M2j29ie3N+mJ39q4WAWYuUuyzdhmzAWYTIMrUowDsn/6bARDEj6HcIsXHZhFMqEqQ==
-  dependencies:
-    "@babel/runtime" "^7.12.5"
-    "@okta/okta-idx-js" "0.24.0"
-    "@peculiar/webcrypto" "1.1.6"
-    Base64 "1.1.0"
-    atob "^2.1.2"
-    btoa "^1.2.1"
-    core-js "^3.6.5"
-    cross-fetch "^3.0.6"
-    js-cookie "2.2.1"
-    node-cache "^5.1.2"
-    p-cancelable "^2.0.0"
-    text-encoding "^0.7.0"
-    tiny-emitter "1.1.0"
-    webcrypto-shim "^0.1.5"
-    xhr2 "0.1.3"
-
 "@okta/okta-auth-js@^6.4.0":
```

Comment:
maybe run add/upgrade again to stay with the latest patch.

Reply:
I'm running tests with beta version, will run again after promotion of auth-js w/ needed patches.